bridge/src/lib.rs

```diff
@@ -111,6 +111,12 @@ pub mod pallet {
 
 		/// Current pallet index defined in runtime
 		type PalletIndex: Get<u8>;
+
+		/// Asset decimal converter
+		type DecimalConverter: DecimalConverter;
+
+		/// AssetId and its decimal pairs
+		type AssetDecimalPairs: Get<Vec<(AssetId, u8)>>;
```

Comment:
It seems only `DecimalConverter` need the pair information, we can remove it from bridge config

Reply:
oh you remind me this, actually I was thinking to make this as a `constant`, so that it will be convenient to check which asset has which decimal, I will add `constant` annotation

Reply:
wait a sec, now I'm thinking that if not removing it from bridge pallet, it means other developers will need to provide it in their runtime config but they might not necessarily to have this decimal pairs defined like I did, and it would be a problem. ideally they will just need to impl the DecimalConverter trait, how to manage the asset and its decimal should be up to them. Ok I will remove it from bridge pallet.